### PR TITLE
Reducers and actions

### PR DESCRIPTION
**Pull request by kazaneza**
### In this pull request 

- Implement Redux Toolkit in a React application by setting up a Redux store and creating a slice to save state and reducers. 
- state management and use store, actions, and reducers in React. 
- Followed Gitflow and no linter errors.

### CODE REVIEW

File: src/redux/books/booksSlice.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      category: 'Fiction',
+    },
+    {
+      item_id: 'item3',
+      title: 'The Selfish Gene',
+      author: 'Richard Dawkins',
+      category: 'Nonfiction',
+    },
+  ],
+};
```
Comment (Owusu-Desmond) [inline src/redux/books/booksSlice.js]: - [ ] Please from the project requirement, you should set the initial state to be an empty array of books. :pray: So make sure that the books array is an empty array :blush:

File: src/redux/books/booksSlice.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, actions) => {
+      const bookData = actions.payload;
+      state.books.push(bookData);
```
Comment (Owusu-Desmond) [inline src/redux/books/booksSlice.js]: - [ ] You have the book you need to add so you need to return a new state by making a copy of the state of the books as well as the book you’re updating

  for instance :point_down: 
  ```javascript
   addBook: (state, action) => {
        state.books = [
          ...state.books, action.payload
        ]
      }
  ```

File: src/quotes.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  },
+  {
+    quote: 'Mathematics is the Queen of the Sciences.',
+    by: 'Carl Friedrich Gauss',
+  },
+  {
+    quote:
+        'But of all the sciences cultivated by mankind, none are more useful than mathematics, to call forth a spirit of enterprise and enquiry.',
+    by: 'Consider Sterry',
+  },
```
Comment (Owusu-Desmond) [inline src/quotes.js]: - [ ] Please why are you creating all these quotes which are not required in this project  :relaxed:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @kazaneza , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — APPROVED** [review]
### Hi @kazaneza,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**